sdk/devcenter/developer-devcenter-rest/package.json

```
@@ -0,0 +1,127 @@
+{
+  "name": "@azure-rest/developer-devcenter",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "version": "1.0.0-beta.1",
+  "description": "",
+  "keywords": [
+    "node",
+    "azure",
+    "cloud",
+    "typescript",
+    "browser",
+    "isomorphic"
+  ],
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "./dist-esm/src/index.js",
+  "types": "./types/developer-devcenter.d.ts",
+  "homepage": "https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/devcenter/developer-devcenter-rest/README.md",
+  "repository": "github:Azure/azure-sdk-for-js",
+  "bugs": {
+    "url": "https://github.com/Azure/azure-sdk-for-js/issues"
+  },
+  "files": [
+    "dist/",
+    "dist-esm/src/",
+    "types/developer-devcenter.d.ts",
+    "README.md",
+    "LICENSE",
+    "review/*"
+  ],
+  "//metadata": {
+    "constantPaths": [
+      {
+        "path": "swagger/README.md",
+        "prefix": "package-version"
+      }
+    ]
+  },
+  "//sampleConfiguration": {
+    "productName": "Azure DevCenter",
+    "productSlugs": [
+      "azure"
+    ],
+    "disableDocsMs": true,
+    "apiRefLink": "https://learn.microsoft.com/azure/dev-box/"
+  },
+  "engines": {
+    "node": ">=12.0.0"
```

We met the ci error https://dev.azure.com/azure-sdk/public/_build/results?buildId=1925208&view=logs&j=efd83a73-1d18-5a8b-eda0-914b9a825166&t=7b68dc2f-1868-549c-7c85-81003212da1d&l=386. 

Could you upgrade the node to 14 manually?
```suggestion
    "node": ">=14.0.0"
```